src/modules/pageDetails.js

```diff
@@ -1,156 +1,33 @@
-            'font-size-medium': column.fontSize === 'medium',
-            'font-size-small': !column.fontSize || column.fontSize === 'small',
-            'font-weight-bold': !column.fontWeight || column.fontWeight === 'bold',
-            'font-weight-normal': column.fontWeight === 'normal',
-            'font-weight-semibold': column.fontWeight === 'semibold',
-        });
-        let accessor, flexBasisInlineStyle, flexGrowInlineStyle, flexShrinkInlineStyle, horizontalSpacingInlineStyle;
-
-        if (_.isString(column.accessor)) {
-            accessor = _.get(data, column.accessor);
-        } else if (_.isFunction(column.accessor)) {
-            accessor = column.accessor(data);
-        }
-
-        if (!!innerContainerKeyNum) {
-            flexBasisInlineStyle = column.flexBasis || 'auto';
-            flexGrowInlineStyle = column.flexGrow || 0;
-            flexShrinkInlineStyle = column.flexShrink || 0;
-            horizontalSpacingInlineStyle = columnProps && columnProps.horizontalSpacing ? `${columnProps.horizontalSpacing}px` : null;
-        }
-
-        return (
-            <div
-                className={containerClasses}
-                key={`page-details-column-key-${innerContainerKeyNum || 0}`}
-                style={Object.assign({}, {
-                    flexBasis: flexBasisInlineStyle,
-                    flexGrow: flexGrowInlineStyle,
-                    flexShrink: flexShrinkInlineStyle,
-                    paddingLeft: horizontalSpacingInlineStyle,
-                    paddingRight: horizontalSpacingInlineStyle,
-                    width: column.width,
-                }, column.style)}
-            >
-                {column.header &&
-                    <Header size="xsmall" style={{ margin: 0 }}>
-                        {column.header}
-                    </Header>
-                }
-
-                <div className={accessorClasses}>
-                    {accessor}
-                </div>
-            </div>
-        );
-    }
-}
-
-PageDetailsColumn.propTypes = {
-    column: PropTypes.oneOfType([
-        PropTypes.array,
-        PropTypes.object,
-    ]).isRequired,
-    columnProps: PropTypes.object,
-    data: PropTypes.object.isRequired,
-};
-
-class PageDetails extends React.PureComponent {
-    render() {
-        const {
-            bleed,
-            className,
-            color,
-            columnProps,
-            columns,
-            data,
-            style,
-        } = this.props;
-        const containerClasses = ClassNames('ui', 'page--details', className, {
-            'page--details-bleed': bleed,
-        });
-        let pageDetailsColumnKeyNum = 1;
-
-        return (
-            <div
-                className={containerClasses}
+const PageDetails = (props) => {
+    const {
+        bleed,
+        className,
+        color,
+        columnProps,
+        columns,
+        data,
+        style,
+    } = props;
+
+    return (
+        <React.Fragment>
```

Comment:
Same comment in drawerDetails file for this file.

Reply:
Fixed